source/opt/block_merge_pass.h

```diff
@@ -38,33 +38,33 @@ class BlockMergePass : public Pass {
  public:
   BlockMergePass();
   const char* name() const override { return "merge-blocks"; }
-  Status Process(ir::IRContext*) override;
-  virtual ir::IRContext::Analysis GetPreservedAnalyses() override {
-    return ir::IRContext::kAnalysisDefUse |
-           ir::IRContext::kAnalysisInstrToBlockMapping |
-           ir::IRContext::kAnalysisDecorations |
-           ir::IRContext::kAnalysisCombinators |
-           ir::IRContext::kAnalysisNameMap;
+  Status Process(opt::IRContext*) override;
+  virtual opt::IRContext::Analysis GetPreservedAnalyses() override {
+    return opt::IRContext::kAnalysisDefUse |
+           opt::IRContext::kAnalysisInstrToBlockMapping |
+           opt::IRContext::kAnalysisDecorations |
+           opt::IRContext::kAnalysisCombinators |
+           opt::IRContext::kAnalysisNameMap;
```

Comment:
All of the `opt::` are no longer needed.  Here and in a lot of other places.  How hard is it to have a tool clean these up?

Reply:
It won't be hard, I just want to do it as a second pass. Tried it as one and the change gets huge and hard to debug.